src/electron/views/device-connect-view/components/device-connect-port-entry.tsx

```diff
@@ -44,7 +44,7 @@ export class DeviceConnectPortEntry extends React.Component<
                 <h3>Android device port number</h3>
                 <MaskedTextField
                     data-automation-id={deviceConnectPortNumberFieldAutomationId}
-                    ariaLabel="Port number"
+                    ariaLabel="Android Device port number e.g. 12345"
```

Comment:
should the aria label match the place holder a little bit more? 
The aria label is using `e.g.` while the place holder uses `Ex:`

Also, this is kind of related to what we were talking offline: if we convert the `<h3>` to a label and use the `for` attribute to associate the label with this input, I think the AT will use that label to announce the input, thus, if we keep this aria label, there is a chance of duplicated text being announced (I'm not 100% sure about that but I think the possibility is real)

That being said, I think this issue (the aria label) and the "let's not use `<h3>` issue should be tackle togheter.

Thoughts?

Reply:
Makes sense to me. I'll see what I can come up with.

Reply:
I dug into the label question a little more, but there's a problem. We're using an office fabric MaskedTextField, which contains 2 divs inside it. HTML doesn't allow a label to apply to something inside a div, so there's no way to bind the current h3 to the edit unless we completely replace the MaskedTextField (which I don't think we want to do at this point). As it turns out, though, MaskedTextField has a label property that does exactly what we want it to do in both NVDA and JAWS.